src/palakSJun20/PrimeNumberVariation.java

```diff
@@ -0,0 +1,84 @@
+/* Program 1: Create a class to satisfy below requirements. 
+                  a) Given number is prime or not. 
+                  b) Print total count of prime numbers between 2->100.
+                  c) Find sum of all prime numbers between 2->100
+                  d) Find avg of prime numbers between 2->100*/
+package palakSJun20;
+
+public class PrimeNumberVariation {
+
+		// Given number is prime or not. 
+		void findGivenNumberPrimeOrNot(int number){
+			boolean flag = false;
+			for (int i = 2; i < number/Math.sqrt(number); i++) {
+				if (number % 2 == 0) {
+					flag = true;
+					break;
+				}
+			}
+			if (flag == true) {
+				System.out.println("Number "+number+ " is not prime");
+			}
+			else 
+				System.out.println("Number "+ number+ " is Prime");
+		}
+		
+		//Print Total count of prime number b/w 2-100
+		int printTotalCountOfPrimeNumber(){
```

Comment:
Why this separate method is required ? You should directly write count and some logic in findGivenNumberPrimeOrNot() method.  Take two instance variable.

Reply:
written sum and average logic in same method where we are counting prime number from 2-100